examples/02-Merlin-Models-and-NVTabular-applying-to-your-own-dataset.ipynb

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
<p>We could add a reference to the Default TAGs : https://github.com/NVIDIA-Merlin/core/blob/main/merlin/schema/tags.py</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
I think we should not reference our source code. We should have a documentation page for it. Let's add it, when documentation is ready